### DISCUSSION TITLE
支持自定义负载均衡策略

### DISCUSSION THREAD

**Discussion by mnixry** [v2fly/v2ray-core#1312] [Ideas]
V2Ray具有一定的[负载均衡功能](https://guide.v2fly.org/app/balance.html#%E9%85%8D%E7%BD%AE), 可以将连接分散到不同的出站接口

但是目前, V2Ray仅有一种[基于轮询的负载均衡策略](https://guide.v2fly.org/app/balance.html#%E5%8E%9F%E7%90%86), 我希望V2Ray-Core能够提供负载均衡策略的可变性(例如简单粗暴的为每个连接随机选择出站)

**Comment by BozhanL** [comment]
https://www.v2fly.org/config/routing.html#balancerobject

**Reply by mnixry** [reply]
感谢, 我瞎比 :rofl:
